S/SCIP_PaPILO/build_tarballs.jl

```diff
@@ -105,7 +105,7 @@ dependencies = [
     Dependency(PackageSpec(name="Bzip2_jll", uuid="6e34b625-4abd-537c-b88f-471c36dfa7a0"); compat="1.0.9"),
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
     Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"); compat="6.2.1"),
-    Dependency(PackageSpec(name="Ipopt_jll", uuid="9cc047cb-c261-5740-88fc-0cf96f7bdcc7"); compat="=300.1400.1400"),
+    Dependency(PackageSpec(name="Ipopt_jll", uuid="9cc047cb-c261-5740-88fc-0cf96f7bdcc7"); compat="=300.1400.1900"),
     Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"); compat="0.3.10"),
```

Comment:
Should we not compile `SCIP` with LBT too?
`Ipopt_jll.jl` is compiled with LBT and is a dependency so let's we have an "hybrid" build now in terms of BLAS / LAPACK.

Reply:
I tried to reproduce the issue with `Downloads` in Julia and didn't succeed.
It could be related that we use Julia 1.7 for the cross-compilation.

I updated the sources to use the github repository.